diff --git a/gn/build/gen.py b/gn/build/gen.py
--- gn/build/gen.py
+++ gn/build/gen.py
@@ -359,12 +359,13 @@
     cflags_cc.extend(['-std=c++14', '-Wno-c++11-narrowing'])
 
     if platform.is_linux():
       ldflags.append('-Wl,--as-needed')
+      ldflags.append('--target ' + os.getenv("CONDA_TOOLCHAIN_BUILD"))
+      cflags.append('--target ' + os.getenv("CONDA_TOOLCHAIN_BUILD"))
+      cflags_cc.append('--target ' + os.getenv("CONDA_TOOLCHAIN_BUILD"))
+      ldflags.append('-static-libstdc++')
 
-      if not options.no_static_libstdcpp:
-        ldflags.append('-static-libstdc++')
-
       # This is needed by libc++.
       libs.append('-ldl')
     elif platform.is_darwin():
       min_mac_version_flag = '-mmacosx-version-min=10.9'